src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -2946,22 +3035,9 @@ function makeWaterfallPlotData_profiledReport(
     return ret;
 }
 
-export function getCnaQueries(utilitiesSelection: ColoringMenuSelection) {
-    const queries: { entrezGeneId: number }[] = [];
-    if (
-        utilitiesSelection.selectedOption !== undefined &&
-        utilitiesSelection.selectedOption.info.entrezGeneId !==
-            NONE_SELECTED_OPTION_NUMERICAL_VALUE &&
-        utilitiesSelection.selectedOption.info.entrezGeneId !== undefined
-    ) {
-        queries.push({
-            entrezGeneId: utilitiesSelection.selectedOption.info.entrezGeneId,
-        });
-    }
-    return _.uniqBy(queries, 'entrezGeneId');
-}
-
-export function getMutationQueries(utilitiesSelection: ColoringMenuSelection) {
+export function getGenomicColoringQueries(
```

Comment:
i'm not sure what genomicColoringuery is.   can we improve name and/or add comment?